inst/odin/lancelot.R

```diff
@@ -1665,10 +1665,43 @@ update(general_tot) <- new_general_tot
 initial(hosp_tot) <- 0
 update(hosp_tot) <- new_ICU_tot + new_general_tot
 
+## cumulative deaths in hospital by age
 initial(D_hosp_tot) <- 0
 delta_D_hosp_tot <- sum(delta_D_hosp)
 update(D_hosp_tot) <- D_hosp_tot + delta_D_hosp_tot
 
+initial(D_hosp_0_49_tot) <- 0
+delta_D_hosp_0_49_tot <- sum(delta_D_hosp[1:10]) + delta_D_hosp[18] * 3 / 8
```

Comment:
It strikes me that the proportion of CHW deaths that we are assuming are for each group is a parameter that's better off not hardcoded. Same for the CHR parameters.

Reply:
And also in hindsight, the ones for pillar 2 by age